Makefile

```diff
@@ -308,7 +308,7 @@ generate-deepcopy: images/dev-env/.dockerbuilt chaos-build
 
 generate: generate-ctrl swagger_spec generate-deepcopy chaos-build
 
-check: generate yaml vet boilerplate lint tidy install.sh fmt
+check: generate yaml vet lint tidy install.sh fmt
```

Comment:
Why remove the prerequisite `boilerplate` from the target `check` here?

Reply:
Oops. It's a misoperation. Added back.